projects/ion/src/lib/dropdown/dropdown.component.spec.ts

```diff
@@ -260,6 +260,60 @@ describe('IonDropdownComponent / Multiple / Clear Options', () => {
   });
 });
 
+describe('IonDropdownComponent / Multiple With Max Length', () => {
+  const optionsWithMultiple = [
+    { label: 'Dog', selected: true },
+    { label: 'Cat', selected: false },
+    { label: 'Horse', selected: false },
+  ];
+
+  const maxSelectedQtd = 1;
+
+  const defaultMultiple = {
+    options: optionsWithMultiple,
+    multiple: true,
+    maxSelected: maxSelectedQtd,
+    selected: {
+      emit: selectEvent,
+    } as SafeAny,
+  };
+
+  it('should not select an option when the defined limit was reached', async () => {
+    await sut(defaultMultiple);
+    const elementToSelect = document.getElementById('option-1');
+    fireEvent.click(elementToSelect);
```

Comment:
Try select another option and check maxSelectedQtd.

Reply:
both on this assertion?

Reply:
Yep, this will ensure that only a option did selected even thought the user try select 2 options.